tgstation.dme

```diff
@@ -2403,7 +2406,6 @@
 #include "code\modules\projectiles\guns\ballistic\rifle.dm"
 #include "code\modules\projectiles\guns\ballistic\shotgun.dm"
 #include "code\modules\projectiles\guns\ballistic\toy.dm"
-#include "code\modules\projectiles\guns\energy\dueling.dm"

```

Comment:
?

Reply:
yeah for some reason byond keeps unchecking that for me, and this time, i didnt catch it